mne/annotations.py

```diff
@@ -537,6 +622,38 @@ def get_duration_from_times(t):
                        orig_time=orig_time)
 
 
+def _is_iso8601(candidate_str):
+    import re
+    ISO8601 = r'^\d{4}-\d{2}-\d{2}[ T]\d{2}:\d{2}:\d{2}\.\d{6}$'
+    return re.compile(ISO8601).match(candidate_str) is not None
+
+
+def _read_annotations_txt_parse_header(fname):
```

Comment:
Why a separate function for this? Will this be used elsewhere? Otherwise, I'd include that in `_read_annotations_txt`.